aea/contracts/base.py

```diff
@@ -192,3 +194,40 @@ def get_state(
         :return: the tx
         """
         raise NotImplementedError
+
+    @staticmethod
```

Comment:
Any code relating to command line interface/cosm* api needs to be placed in `crypto/cosmos.py` CosmosApi